src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -527,7 +529,8 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 			this.table.rerenderGrid();
 		}));
 		if (this.enableFilteringFeature) {
-			this.filterPlugin = new HeaderFilter(this.contextViewService);
+			this.filterPlugin = new HeaderFilter(this.contextViewService, this.notificationService);
```

Comment:
Could also consider just using the instantiation service instead of having to pass in each of these separate services